client/js/components/user/orgaDataEntry/OrganisationCompetence.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <fieldset
+    v-if="hasPermission('field_organisation_competence')"
```

Comment:
I think we should move the permission check outside of the component:

```
<organisation-competence
  v-if="hasPermission('field_organisation_competence')"
  :organisation="organisation" />
```

Reply:
Why would you prefer it this way?

Reply:
If we check the permission outside, the component is not rendered at all if the permission is `false`. We could even import it dynamically based on the permission. In that case, the component will not be imported or initialized if the permission is `false`.

Reply:
True. But I also like to have it in the component, because then I don't have to worry about permission checks when importing the component (and can't forget it). Done in [2aa9cdc](https://github.com/demos-europe/demosplan-core/pull/4237/commits/2aa9cdc387a0acf1c5da018a1b382877577924f6)

Reply:
That's also a good point. Maybe we should discuss it in the FE team?